src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/.template.config.src/template.json

```diff
@@ -5,36 +5,40 @@
     "Blazor",
     "Hosted"
```

Comment:
I'm not sure how classifications are used, but should "Hosted" be replaced with "WebAssembly"?

Reply:
ok